doc/introduction/photonic_hardware.rst

```diff
@@ -0,0 +1,95 @@
+Photonic hardware
+=================
+
+Using Strawberry Fields, you can submit quantum programs
+to be executed on photonic hardware using the Xanadu cloud platform.
+
+
+Configuring your account
+------------------------
+
+Before using the Xanadu cloud platform, you will need to configure your credentials. This
+can be done using several methods:
+
+* **Configuration file (recommended)**. Use the Strawberry Fields :func:`~.store_account` function,
+
+  >>> sf.store_account("AUTHENTICATION_TOKEN")
+
+  or the ``sf configure`` :doc:`command line interface </code/sf_cli>`,
+
+  .. code-block:: bash
+
+      $ sf configure --token AUTHENTICATION_TOKEN
+
+  to store a configuration file containing your credentials. Replace
+  ``AUTHENTICATION_TOKEN`` above with your Xanadu cloud access token.
+
+* **Environment variables**. Set the ``SF_API_AUTHENTICATION_TOKEN`` environment variable
+  with your API token.
+
+* **Keyword argument**. Pass the keyword argument ``token`` when initializing the
+  :class:`~.RemoteEngine`.
+
+.. seealso::
+
+    For more details on configuring Strawberry Fields for cloud access, including
+    creating local per-project configuration files, see the :doc:`Advanced configuration options </code/sf_configuration>`, and :func:`~.store_account` function.
+
+
+Verifying your connection
+-------------------------
+
+To verify that your account credentials correctly authenticate against the cloud
+platform, the :func:`~.ping` function can be used from within Python,
+
+>>> sf.cli.ping()
+You have successfully authenticated to the platform!
+
+or the ``ping`` flag can be used on the command line:
+
+.. code-block:: bash
+
+    $ sf --ping
+    You have successfully authenticated to the platform!
+
+.. seealso::
+
+   :doc:`command line interface </code/sf_cli>`
```

Comment:
Maybe this makes more sense to be combined with the seealso above, e.g. by adding a sentence like:
"Further details on the Strawberry Fields command line interface can be found in :doc:`command line interface </code/sf_cli>`."